SparkPost/2web2ui
# TR-3040 - Resolves (UI) Templates validator in AB testing possibly too strict

[TR-3040](https://sparkpost.atlassian.net/browse/TR-3040)

### What Changed

- Updates validation function to use `Array.prototype.some()` instead of `.includes()` to check for matching `id` values for the selected template vs. the list of templates

### How To Test

- Go to `/ab-testing/create` and try to create an A/B testing
- Verify that you cannot create an A/B test if it is not available to that particular subaccount
- Verify that you cannot create an A/B test if the template is not available to the primary account when no subaccount is selected
- Verify that you _can_ create a template when a matching ID is present in the filtered list of templates passed to the validation helper

### To Do
